Code/Editor/CryEdit.cpp

```diff
@@ -1903,8 +1903,12 @@ void CCryEditApp::WriteConfig()
 // App command to run the dialog
 void CCryEditApp::OnAppAbout()
 {
-    CAboutDialog aboutDlg(FormatVersion(m_pEditor->GetFileVersion()), FormatRichTextCopyrightNotice());
-    aboutDlg.exec();
+    auto* dialog = new CAboutDialog(FormatVersion(m_pEditor->GetFileVersion()), FormatRichTextCopyrightNotice());
+    auto mainWindow = MainWindow::instance();
+    auto geo = dialog->geometry();
+    geo.moveCenter(mainWindow->mapToGlobal(mainWindow->geometry().center()));
+    dialog->setGeometry(geo);
+    dialog->show();
```

Comment:
Need to set the `Qt::WA_DeleteOnClose` attribute on the `CAboutDialog` to make this change, otherwise this will cause a memory leak since the dialog won't actually be deleted anymore.